InventoryManagementSystem/Domain/Main/Utilities.cs

```diff
@@ -113,11 +112,23 @@ private static void SearchProduct()
             }
         }
 
-        internal static void InitializeStock()
+        internal static void InitializeStock(string databaseType)
         {
-            inventory.AddProduct("Sugar", new Price() { ItemPrice = 10, Currency = Currency.Euro }, 100);
-            inventory.AddProduct("Cake decorations", new Price() { ItemPrice = 8, Currency = Currency.Euro }, 20);
-            inventory.AddProduct("Strawberry", new Price() { ItemPrice = 3, Currency = Currency.Euro }, 10);
+            IProductRepository productRepository;
+
+            if (databaseType == "MongoDB")
+            {
+                productRepository = new MongoDbProductRepository(); 
+            }
+            else
```

Comment:
could be not sqlserver, so lets check here if it's really sqlserver, if it's something, lets throw an nonimplemented exception.

Reply:
also, there's a design pattern that you can use here instead :D